Studio-Lovelies/GG-JointJustice-Unity
# Feature/input system

## Summary

Adds the Input System package, along with code to create an Input Actions object (PlayerInput.cs)
Unity's new Input System will make supporting other devices easier (gamepads/mobile), and allow controls to be configurable at runtime.
Also updates the .gitignore to ignore Rider's cache directory.

## Before/after screenshots and/or animated gif


## Testing instructions

Check scenes with event systems run as expected. Event systems may need to be updated to use the new input system.

## Additional information

- `[ ]` Changes UI
- `[x]` Introduces new feature
- `[ ]` Removes existing feature
- `[ ]` Has associated resource:
